### DISCUSSION TITLE
Set 'Allow-Control-Allow-Origin' to '*'?

### DISCUSSION THREAD

**Discussion by SanokKule** [rejetto/hfs#141] [Ideas]
I'm trying to load my font in Better Discord from my hfs server, but it throws:
`Access to font at 'font_url' from origin 'https://ptb.discord.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.`
Maybe there should be an option to allow changing it, or just set `Access-Control-Allow-Origin`  in Response Header to `*` by default?

**Comment by rejetto** [comment]
i will address this asap

**Comment by rejetto** [comment]
please, try the new plugin i just created
Admin panel > plugins > search online > cors

**Reply by SanokKule** [reply]
Now it works, i don't see any problems
